assets/js/listeners.js

```diff
@@ -11,10 +11,17 @@ window.addEventListener("moon:update-select", (event) => {
 });
 
 window.addEventListener("moon:update-switch", (event) => {
-  var checkbox = event.target;
+  var radio_true = event.target;
   var detail = event.detail;
-  checkbox.checked = detail.checked;
-  checkbox.dispatchEvent(new Event("input", { bubbles: true }));
+  var switch_id = detail.switch_id;
+  var radio_false = document.getElementById(switch_id + "_radio_false");
+  if (detail.checked) {
```

Comment:
I think, because it is radio in group, if one is selected, other should be automatically unselected?